pkg/mongoproxy/plugins/schema/schema.go

```diff
@@ -38,7 +38,8 @@ var (
 )
 
 const (
-	Name = "schema"
+	Name                 = "schema"
+	ENFORCESCHEMALOGONLY = true
```

Comment:
can we pass it as service parameter, such that we change parameter instead of changing code to control this behavior?

Reply:
updated